client/src/components/LoginInput.js

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+const LoginInput = styled.input`
```

Comment:
What makes this `input` a `LoginInput`? Is there something special about it, that it has to be used in the login context like a special icon or placeholder?
If not, just call it `Input` ;).

Actually, it is an input element with basic styling.

Reply:
Yes, this input will only be used in the login/signup context. Since the input that I'll use later on will look completely different, I would prefer to separate them.